melodies_monet/driver.py

```diff
@@ -148,41 +148,76 @@ def open_obs(self, time_interval=None):
         """
         from glob import glob
         from numpy import sort
-        from . import tutorial
-
-        if self.file.startswith("example:"):
-            example_id = ":".join(s.strip() for s in self.file.split(":")[1:])
-            files = [tutorial.fetch_example(example_id)]
-        else:
-            files = sort(glob(self.file))
-
-        assert len(files) >= 1, "need at least one"
-
-        _, extension = os.path.splitext(files[0])
+ 
         try:
-            if extension in {'.nc', '.ncf', '.netcdf', '.nc4'}:
-                if len(files) > 1:
-                    self.obj = xr.open_mfdataset(files)
-                else:
-                    self.obj = xr.open_dataset(files[0])
-            elif extension in ['.ict', '.icarrt']:
-                assert len(files) == 1, "monetio.icarrt.add_data can only read one file"
-                self.obj = mio.icarrt.add_data(files[0])
-            else:
-                raise ValueError(f'extension {extension!r} currently unsupported')
-        except Exception as e:
-            print('something happened opening file:', e)
-            return
+            if os.path.isfile(self.file):
+                _, extension = os.path.splitext(self.file)
+                if extension in ['.nc', '.ncf', '.netcdf', '.nc4']:
+                    if len(glob(self.file)) > 1:
+                        self.obj = xr.open_mfdataset(sort(glob(self.file)))
+                    self.obj = xr.open_dataset(self.file)
+                elif extension in ['.ict', '.icarrt']:
+                    self.obj = mio.icarrt.add_data(self.file)
+                self.mask_and_scale()  # mask and scale values from the control values
+        except ValueError:
+            print('something happened opening file')
+            
+    def open_sat_obs(self,time_interval=None):
+        """Methods to opens satellite data observations. 
+        Uses in-house python code to open and load observations.
+        Alternatively may use the satpy reader.
 
-        self.mask_and_scale()  # mask and scale values from the control values
-        self.filter_obs()
+        Returns
+        -------
+        type
```

Comment:
```suggestion
        None
```
This method doesn't return anything.

Reply:
@dwfncar